<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomPointDock</class>
 <widget class="QWidget" name="CustomPointDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>563</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="3">
        <widget class="ResizableTextEdit" name="teComment">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="lName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lPositionXLogical">
         <property name="text">
          <string>x:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="KComboBox" name="cbPositionX"/>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbPositionX">
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="lComment">
         <property name="text">
          <string>Comment:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lPositionY">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Vertical position relative to label's parent</string>
         </property>
         <property name="text">
          <string>y:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="3">
        <widget class="QComboBox" name="cbPlotRanges"/>
       </item>
       <item row="7" column="3">
        <widget class="QLineEdit" name="lePositionXLogical"/>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="leName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lBindLogicalPos">
         <property name="text">
          <string>Logical coordinates</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="lPosition">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="lPositionXLogicalDateTime">
         <property name="text">
          <string>x:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="3">
        <widget class="QLabel" name="lPlotRange">
         <property name="text">
          <string>Plot range:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="KComboBox" name="cbPositionY"/>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sbPositionY">
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lPositionX">
         <property name="toolTip">
          <string>Horizontal position relative to label's parent</string>
         </property>
         <property name="text">
          <string>x:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lPositionYLogical">
         <property name="text">
          <string>y:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>694</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="3">
        <widget class="QLineEdit" name="lePositionYLogical"/>
       </item>
       <item row="6" column="3">
        <widget class="QDateTimeEdit" name="dtePositionXLogical">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="3">
        <widget class="QCheckBox" name="chkVisible">
         <property name="text">
          <string>Visible</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QCheckBox" name="chbBindLogicalPos">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lPositionYLogicalDateTime">
         <property name="text">
          <string>y:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QDateTimeEdit" name="dtePositionYLogical"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSymbol">
      <attribute name="title">
       <string>Symbol</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>src/kdefrontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>teComment</tabstop>
  <tabstop>chbBindLogicalPos</tabstop>
  <tabstop>cbPositionX</tabstop>
  <tabstop>sbPositionX</tabstop>
  <tabstop>dtePositionXLogical</tabstop>
  <tabstop>lePositionXLogical</tabstop>
  <tabstop>cbPositionY</tabstop>
  <tabstop>sbPositionY</tabstop>
  <tabstop>lePositionYLogical</tabstop>
  <tabstop>cbPlotRanges</tabstop>
  <tabstop>chkVisible</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
